.buildkite/longruns/pipeline.yml

```diff
@@ -39,14 +39,8 @@ steps:
       JULIA_NUM_PRECOMPILE_TASKS: 8
       JULIA_MAX_NUM_PRECOMPILE_FILES: 50
 
-    concurrency: 1
```

Comment:
Is it okay to remove this? It seemed like it was needed in [#709](https://github.com/CliMA/ClimaCoupler.jl/pull/709)

Reply:
This depends on how often we want to run longruns. Sometimes (though not always) the pipeline init fails if two builds are triggered at the same time (i.e., the depot is updated by both at the same time). Setting up these concurrency groups forces one of the pipeline `init`s to wait, until the first one is finished. So in our normal CI, where we trigger concurrent builds every day, this is needed. In the longruns this can still be useful (on the offchance that we're triggering several runs), but this also forces the `new-central` and `clima` inits to wait for each other each time (adding 10 mins or more to each build). Since we don't trigger the longruns too frequently, I think it's more useful to be able to trigger both pipelines simultaneously. There might be a way to set up concurrency groups to separate clima and new-central environment inits, but not sure how complicated this would be. @Sbozzolo do you know?

Reply:
Longruns are not using a dedicated depot and different builds will have their personal depot (the different compared to normal runs is that we don't set JULIA_DEPOT_PATH). So, I don't think that this concurrency group is needed.